packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -1400,14 +1411,28 @@ export class ColorPicker
     context.arc(x, y, radius, startAngle, endAngle);
     context.fillStyle = "#fff";
     context.fill();
+
     context.strokeStyle = "rgba(0,0,0,0.3)";
     context.lineWidth = outlineWidth;
     context.stroke();
 
+    const pattern = context.createPattern(this.getCheckeredBackgroundPattern(), "repeat");
+
+    context.beginPath();
+    context.arc(x, y, radius - 3, startAngle, endAngle);
+    context.fillStyle = pattern;
+    context.fill();
+
+    context.globalCompositeOperation = "source-atop";
```

Comment:
Noice! I was not aware of this prop. Will need to add it to the toolbox. ✨🧰✨

Reply:
Yeah, it's layering operations, photoshop style. Here is a playground: https://www.w3schools.com/jsref/playcanvas.php?filename=playcanvas_globalcompop&preval=source-atop